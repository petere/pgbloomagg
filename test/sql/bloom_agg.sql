CREATE TABLE test (a int, b text);

SELECT bloom_agg(b, 16) FROM test;

INSERT INTO test VALUES (1, 'one');

SELECT bloom_agg(b, 16) FROM test;

INSERT INTO test VALUES (2, 'two');
INSERT INTO test VALUES (3, 'three');
INSERT INTO test VALUES (4, 'four');
INSERT INTO test VALUES (5, 'five');
INSERT INTO test VALUES (6, 'six');

SELECT bloom_agg(b, 16) FROM test;
