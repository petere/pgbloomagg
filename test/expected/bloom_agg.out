CREATE TABLE test (a int, b text);
SELECT bloom_agg(b, 16) FROM test;
 bloom_agg 
-----------
 
(1 row)

INSERT INTO test VALUES (1, 'one');
SELECT bloom_agg(b, 16) FROM test;
    bloom_agg     
------------------
 0000000000000100
(1 row)

INSERT INTO test VALUES (2, 'two');
INSERT INTO test VALUES (3, 'three');
INSERT INTO test VALUES (4, 'four');
INSERT INTO test VALUES (5, 'five');
INSERT INTO test VALUES (6, 'six');
SELECT bloom_agg(b, 16) FROM test;
    bloom_agg     
------------------
 0110010000100110
(1 row)

